doc/guide/caboose.html

```diff
@@ -0,0 +1,646 @@
+<!DOCTYPE html>
```

Comment:
Did you intend to check this in? I was briefly shocked by the diffstat

Reply:
agh, whoops --- i staged the wrong file by mistake. will fix!